packages/chain-adapters/src/utils/index.ts

```diff
@@ -38,7 +38,12 @@ export const chainIdToChainLabel = (chainId: ChainId): string => {
 
   switch (chainNamespace) {
     case CHAIN_NAMESPACE.Bitcoin:
-      return 'bitcoin'
+      switch (chainReference) {
+        case CHAIN_REFERENCE.DogecoinMainnet:
+          return 'dogecoin'
+        default:
```

Comment:
prefer exhaustive cases for valid chain references and the default to throw invalid